docs/docs/publishing/simulating-multiplayer.md

```diff
@@ -12,4 +12,10 @@ To simulate the multiplayer experience during development, Rune runs many instan
 
 ## Run on Your Phone
 
-You can also test your game on your phone. Just connect to the URL provided by your development tool and you'll see a development UI for testing the multiplayer experience. You can also test how your game works on a phone by using Chrome Developer Tools.  
+You can also test your game on your phone. Just connect to the URL provided by your development tool and you'll see a development UI for testing the multiplayer experience. You can also test how your game works on a phone by using Chrome Developer Tools.
+
+## FAQ
```

Comment:
The FAQ question is separate, see the `faq.md` file :)

Reply:
I.e. we should add it both in this section and in the FAQ.

Reply:
Fixed in https://github.com/rune/rune-games-sdk/pull/192/commits/e8e5b28c7b3ddd56c230b463216d2a2e0ee76153